lib/dry/validation/result.rb

```diff
@@ -81,3 +81,11 @@ def result_ast
     end
   end
 end
+
+begin
```

Comment:
Then using `Dry::Validation.load_extensions` does not suppose using `rescue LoadError`, am I right?